dev/NumberBox/NumberBox.idl

```diff
@@ -44,28 +45,35 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     Double Value;
 
     [MUX_DEFAULT_VALUE("1")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     Double StepFrequency;
 
-    Object Header;
-    Windows.UI.Xaml.DataTemplate HeaderTemplate;
-
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     String Text;
 
+    // TextBox properties
+    Object Header;
+    Windows.UI.Xaml.DataTemplate HeaderTemplate;
     String PlaceholderText;
+    Windows.UI.Xaml.Controls.Primitives.FlyoutBase SelectionFlyout;
+    Windows.UI.Xaml.Media.SolidColorBrush SelectionHighlightColor;
+    Windows.UI.Xaml.TextReadingOrder TextReadingOrder;
+    Boolean PreventKeyboardDisplayOnProgrammaticFocus;
+    Object Description;
 
```

Comment:
If this is Object it should probably have a Template. So maybe it should just be String?

Reply:
It's defined on TextBox as an Object, and I'm just passing it along, so I think it has to stay an Object. The TextBox template has it as the content of a ContentPresenter with no template. 🤷‍♀